templates/layout.html

```diff
@@ -73,8 +73,7 @@ <h1>Settings</h1>
       <footer class="grid-item">
         <div class="links">
           <a href="/api/docs">API</a>
-          <a href="https://github.com/Rapptz/jimaku">GitHub</a>
-          <a href="https://discord.gg/xtEBbH75ph">Discord</a>
+          <a href="/contact">Contact Us</a>
```

Comment:
I'd rather this just get appended rather than replaced.
```suggestion
          <a href="/contact">Contact</a>
          <a href="https://github.com/Rapptz/jimaku">GitHub</a>
          <a href="https://discord.gg/xtEBbH75ph">Discord</a>
```

Reply:
These links are duplicates. I'd rather suggest not repeating ourselves more than necessary.

Reply:
The fact it's duplicated doesn't matter to me.